conf/msp/msp.yaml

```diff
@@ -24,6 +24,17 @@ elasticsearch:
   username: "${ES_SECURITY_USERNAME}"
   password: "${ES_SECURITY_PASSWORD}"
 
+elasticsearch@logs:
```

Comment:
msp-service should not directly connect to es to query logs. It is better to query logs through the grpc API of monitor-service

Reply:
it's temporary code to compitible with the old log-analysis query logic which not suit to integrate with the logic in monitor, as we turn on the new union es storage, this logic will be removed.